README.md

```diff
@@ -86,6 +165,9 @@ trainer.register_validation_hook(validation_dataset)
 trainer.train(train_dataset)
 ```
 
-## padertorch.Configurable
+See the [trainer](padertorch/train/trainer.py#L40) for an explanation of its signature.
+If you want to use `pt.io.get_new_storage_dir` to manage your experiments, you have to define an environment variable `STORAGE_ROOT` which points to the path where all your experiments will be stored, i.e., in the example above, a new directory under `$STORAGE_ROOT/my_experiment_1` will be created.
```

Comment:
I think we should advertise using either get_new_storage_dir or use get_new_subdir directly because in the second case you do not have to define STORAGE_ROOT and can still use all other benefits.